scoper.inc.php

```diff
@@ -76,7 +76,8 @@
 		Finder::create()
 			->files()
 			->ignoreVCS( true )
-			->name( "#($google_services)\.php#" )
+			->name( "#^($google_services)\.php$#" )
+			->depth( '== 0' )
```

Comment:
I made these changes to get rid of some unused classes that had been accidentally included so far, as I just noticed. It's unrelated, but was such a small change not worth making into an issue.

The following classes are now no longer included:
* `Google_Service_Apigee_Resource_OrganizationsEnvironmentsAnalytics`
* `Google_Service_FirebaseDynamicLinksAPI_GooglePlayAnalytics`
* `Google_Service_FirebaseDynamicLinksAPI_ITunesConnectAnalytics`
* `Google_Service_FirebaseDynamicLinks_GooglePlayAnalytics`
* `Google_Service_FirebaseDynamicLinks_ITunesConnectAnalytics`
* `Google_Service_Partners_Analytics`
* `Google_Service_Partners_Resource_Analytics`
* `Google_Service_YouTubeAnalytics`

These had been included simply because their filenames contained `Analytics.php` - while not actually being relevant.

Reply:
@aaemnnosttv Can you check whether this looks good to you?

Reply:
LGTM! 👍